[IMP] Inventory: Update dropshipping (#3607)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst

=============================================================
Use dropshipping to ship directly from suppliers to customers
=============================================================

Dropshipping is an order fulfillment strategy that allows sellers to have items shipped directly
from suppliers to customers. Normally, a seller purchases a product from a supplier, stores it in
their inventory, and ships it to the end customer once an order is placed. With dropshipping, the
supplier is responsible for storing and shipping the item. This benefits the seller by reducing
inventory costs, including the price of operating warehouses.

Configure products to be dropshipped
====================================

> To use dropshipping as a fulfillment strategy, navigate to the :guilabel:`Purchase` app and
> select :menuselection:`Configuration --> Settings`. Under the :guilabel:`Logistics` heading, click
> the :guilabel:`Dropshipping` checkbox.

Reviewer: 100th char breaks
Also need to **Save** settings.
```suggestion
To use dropshipping as a fulfillment strategy, navigate to the :guilabel:`Purchase` app and select
:menuselection:`Configuration --> Settings`. Under the :guilabel:`Logistics` heading, click the
:guilabel:`Dropshipping` checkbox and :guilabel:`Save` to finish.
```

Next, go to the :guilabel:`Sales` app, click :menuselection:`Products --> Products` and choose an

> existing product or select :guilabel:`Create` to configure a new one. On the product page, make sure
> that the :guilabel:`Can be Sold` and :guilabel:`Can be Purchased` checkboxes are enabled.

Reviewer: These are fine, so more of a FYI: 
Can use the guilabel as an action verb in a sentence just to cut down on wordiness 🧠 
```suggestion
existing product or :guilabel:`Create` a new one. On the product page, make sure that the
:guilabel:`Can be Sold` and :guilabel:`Can be Purchased` checkboxes are enabled.
```

.. image:: dropshipping/sold_purchased_checkboxes.png
    :align: center
    :alt: Enable the "Can be Sold" and "Can be Purchased" checkboxes on the product form.

Click on the :guilabel:`Purchase` tab and specify a vendor and the price that they sell the product
for. Multiple vendors can be added, but the vendor at the top of the list will be the one
automatically selected for purchase orders.

.. image:: dropshipping/product_vendor_config.png
    :align: center
    :alt: The product form with a vendor specified.

Finally, select the :guilabel:`Inventory` tab and enable the :guilabel:`Dropship` checkbox in the
*Routes* section.

.. image:: dropshipping/enable_dropship_route.png
    :align: center
    :alt: Enable the Dropship option in the product inventory tab.

Fulfill orders using dropshipping
=================================

When a customer completes an online purchase for a dropshipped product, a sales order will be

> automatically created with an associated request for quotation (RFQ) for the supplier. Sales orders

Reviewer 2: Add the :abbr:`RFQ (Request for Quotation)` in the parenthesis and capitalize Request & Quotation

for the supplier. Sales orders can be viewed in the :guilabel:`Sales` app by selecting
:menuselection:`Orders --> Orders`. Click the :guilabel:`Purchase` smart button at the top right of
a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.

> .. image:: dropshipping/dropship-sales-order.png

Reviewer: Similar. Could crop this more but it's ok for now.

   :align: center
   :alt: A dropship sales order with the Purchase smart button in the top right corner.

.. tip::
   Dropship sales orders can also be created manually through the :guilabel:`Sales` app by selecting

>    :menuselection:`Orders --> Orders --> Create` and adding a product configured for dropshipping.

Reviewer: Would move this above the image, on line 50, so it's in context to the text; it's good practice to try and end an idea or instruction on an image since it's supplemental.

Once the :abbr:`RFQ (Request for Quotation)` is confirmed, it becomes a purchase order and a
dropship receipt is created and linked to it. The receipt can be viewed by clicking the

> :guilabel:`Receipt` smart button at the top right of the purchase order.

Reviewer: Slightly more navigational context...
You'd be surprised how often smart buttons get missed 😅 
```suggestion
:guilabel:`Receipt` smart button in the top right corner of the purchase order form.
```

> .. image:: dropshipping/dropship-purchase-order.png

Reviewer: Could crop this more but it's ok for now.

    :align: center
    :alt: A dropship purchase order with the Receipt smart button in the top right corner.

> The dropship receipt will list the supplier in the *Receive From* section and the customer in the

Reviewer 2: :guilabel: Receive From

in the :guilabel:`Destination Location` section. Upon delivery of the product to the customer, click
on the :guilabel:`Validate` button at the top left of the dropship receipt to confirm the delivered
quantity.

> .. image:: dropshipping/validate-dropship-receipt.png

Reviewer: Also here: could crop this more but it's ok for now.

    :align: center
    :alt: Validate the dropship receipt after delivery.

To view all dropship orders, simply navigate to the :guilabel:`Inventory` dashboard and click the

> green button on the *Dropship* card.

Reviewer 2: :guilabel: Dropship

> green button on the :guilabel:`Dropship` card.

Reviewer: I've seen it written like this on other docs.
Also teal > green (since it's a main Odoo color) 😉 
```suggestion
teal :guilabel:`# TO PROCESS` button on the :guilabel:`Dropship` card.
```
